tests/Microsoft.Bot.Streaming.Tests/Payloads/RequestManagerTests.cs

```diff
@@ -152,10 +151,9 @@ public void RequestManager_GetResponse_ThrowsOnCancelledTask()
             Task.WaitAll(
                 Task.Run(async () =>
                 {
-                    await Assert.ThrowsExceptionAsync<TaskCanceledException>(async () =>
+                    await Assert.ThrowsAsync<TaskCanceledException>(async () =>
```

Comment:
This can be set in one line, and no need to have the variable `r` declared, same for the `await` keyword, (following with more cases).

Reply:
This doesn't work in all the tests, so let's just leave it as it is for now. Thanks!

Reply:
Yeah, I thought it could work, but it seems that it behaves different in C# compared to JavaScript. Thanks anyways @ceciliaavila.